WebDriverAgentTests/IntegrationTests/FBElementScreenshotTests.m

```diff
@@ -48,6 +49,11 @@ - (void)testElementScreenshot
   UIImage *image = [UIImage imageWithData:screenshotData];
   XCTAssertNotNil(image);
   XCTAssertTrue(image.size.width > image.size.height);
+
+  XCUIScreen *mainScreen = XCUIScreen.mainScreen;
+  UIImage *buttonScreenshot = button.screenshot.image;
+  XCTAssertEqual(buttonScreenshot.size.height * mainScreen.scale, image.size.height);
```

Comment:
These are float values. Please use https://developer.apple.com/documentation/xctest/xctassertequalwithaccuracy to compare them

Reply:
same below